app/Services/TransactionRulePipeline.php

```
@@ -35,12 +35,14 @@ public function process(Transaction $transaction): Transaction
             })
             ->toArray();
 
-        return app(Pipeline::class)
+        $result = app(Pipeline::class)
             ->send($transaction)
             ->through($rules)
             ->then(function (Transaction $transaction) {
                 return $transaction;
             });
+        $result->save();
```

PHPStan (2.1.15)

Cannot call method save() on mixed.


(method.nonObject)